projects/ardin_mb/README.md

```diff
@@ -1,4 +1,4 @@
-# Arden et al. Model of navigation
+# Ardin et al. Model of navigation
```

Comment:
:blush:

Reply:
(that wasn't supposed to be a happy blush, more a shameful one but github didn't oblige)